bot/cogs/moderation.py

```diff
@@ -693,19 +653,14 @@ def mod_log(self) -> ModLog:
     @with_role(*MODERATION_ROLES)
     @command(hidden=True, aliases=["shadowtempban, stempban"])
     async def shadow_tempban(
-        self,
-        ctx: Context,
-        user: UserTypes,
-        duration: ExpirationDate,
-        *,
-        reason: str = None
+        self, ctx: Context, user: UserTypes, duration: ExpirationDate, *, reason: str = None
     ) -> None:
         """
-        Create a temporary ban infraction in the database for a user.
+        Create a temporary ban infraction for a user with the provided reason.
 
-        **`user`:** Accepts user mention, ID, etc.
-        **`duration`:** The duration for the temporary ban infraction
-        **`reason`:** The reason for the temporary ban.
+        Duration strings are parsed per: http://strftime.org/
+
+        This does not send the user a notification.
```

Comment:
This should probably be put in the summary so it can be distinguished from the regular counterpart more easily.

Reply:
I don't see this as particularly unclear, and the resulting one-liner for many of them don't fit as single line docstrings.